python/build-wheel-manylinux2014.sh

```diff
@@ -95,7 +95,10 @@ done
 # hack, we should use auditwheel instead.
 for path in .whl/*.whl; do
   if [ -f "${path}" ]; then
-    mv "${path}" "${path//linux/manylinux2014}"
+    out="${path//-linux/-manylinux2014}"
+    if [ "$out" != "$path" ]; then
+        mv "${path}" "${out}"
```

Comment:
why is this necessary?

Reply:
Updated the PR description to explain. The reason for the  `"$out" != "$path"` is that `mv` fails if src & dst are the same location.